test/loginTest.test.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
+var __importDefault = (this && this.__importDefault) || function (mod) {
+    return (mod && mod.__esModule) ? mod : { "default": mod };
+};
+Object.defineProperty(exports, "__esModule", { value: true });
+const chai_1 = __importDefault(require("chai"));
+const userRoute_1 = __importDefault(require("../src/routers/userRoute"));
+const chai_http_1 = __importDefault(require("chai-http"));
+const app_1 = __importDefault(require("../src/app"));
+//assertion style
+chai_1.default.use(chai_http_1.default);
+chai_1.default.should();
+describe('testing Login API', () => {
+    describe("test if user credentials are valid", () => {
+        // in case of valid credentials and user is found in database
+        it("it should return status code of 200", (done) => {
+            const user = {
+                email: "ruthuwmahoro2@gmail.com",
+                password: "Password2@!"
+            };
+            chai_1.default
+                .request(app_1.default)
+                .post('/api/user/login')
+                .send(user)
+                .end((err, response) => {
+                response.should.have.status(200);
+                response.should.to.have.property('');
+                response.body.should.have.keys('status', 'token');
+                done();
+            });
+            should(res).to.have.status(400);
```

Comment:
why do we have testing codes into js if it's being compiled add them into gitingore for better

Reply:
solved